archinstall/locales/base.pot

```diff
@@ -784,3 +784,6 @@ msgstr ""
 
 msgid "Delete subvolume"
 msgstr ""
+
+msgid "Configured {} interfaces"
+msgstr ""
```

Comment:
Why does this PR require a new `msgid` in all the languages? Was this something that was forgotten with a previous commit?

Reply:
I'm not sure why. I used Poedit to translate the text and followed the steps provided in the README file within locales. 

I did notice this `msgid` suddenly became a requirement, though I presumed it was the work of the `./locales_generator.sh`.